library/src/main/java/com/datatorrent/lib/io/fs/FileSplitterInput.java

```diff
@@ -96,7 +96,7 @@ public FileSplitterInput()
   {
     super();
     currentWindowRecoveryState = Lists.newLinkedList();
-    idempotentStorageManager = new IdempotentStorageManager.FSIdempotentStorageManager();
+    idempotentStorageManager = new IdempotentStorageManager.NoopIdempotentStorageManager();
```

Comment:
I understand why we don't want to use FSIdempotentStorageManger in FileSplitter as FileSplitter is more generic than FS. However I think setting Noop disables idempotency by default. How about letting the user knowingly make that choice by not creating it in the operator and set to @NotNull so the user will have to set it.